text/chapter5.md

```diff
@@ -466,7 +466,7 @@ This can be useful for improving readability of code in some circumstances.
  ## Exercises
 
  1. (Easy) Construct a value of type `Shape` which represents a circle centered at the origin with radius `10.0`.
- 1. (Medium) Write a function from `Shape`s to `Shape`s, which scales its argument by a factor of `2.0`, center the origin.
+ 1. (Medium) Write a function from `Shape`s to `Shape`s, which scales its argument by a factor of `2.0`, and centers it at the origin.
```

Comment:
This felt a little ambiguous to me, so I updated the text slightly to match what I interpreted the exercise to be.

Reply:
Feel free to change any of the exercise text to improve clarity.